File: internship_backend_nov24/jury_solutions/cs/b.cs
using System;
using System.Linq;


class Program
{
    static void Main()
    {
        Solve();
    }

    static void Solve()
    {
        int n = int.Parse(Console.ReadLine());
        var throwsX = 0L;
        var throwsY = 0L;

        for (int i = 0; i < n; i++)
        {
            var longs = Console.ReadLine().Split().Select(long.Parse).ToArray();
            throwsX -= longs[0];
            throwsY -= longs[1];
        }

        for (int i = 0; i < n; i++)
        {
            var longs = Console.ReadLine().Split().Select(long.Parse).ToArray();
            throwsX += longs[0];
            throwsY += longs[1];
        }

        Console.WriteLine($"{throwsX / n} {throwsY / n}");
    }
}

File: internship_backend_nov24/jury_solutions/cpp/b.cpp
#include <bits/stdc++.h>

int main() {
    int n;
    std::cin >> n;
    long long throwsX = 0L;
    long long throwsY = 0L;

    for (int i = 0; i < n; i++) {
        long long x, y;
        std::cin >> x >> y;
        throwsX -= x;
        throwsY -= y;
    }

    for (int i = 0; i < n; i++) {
        long long x, y;
        std::cin >> x >> y;
        throwsX += x;
        throwsY += y;
    }

    std::cout << throwsX / n << ' ' << throwsY / n;
    return 0;
}